mat/vector_test.go

```diff
@@ -561,3 +561,16 @@ func randVecDense(size, inc int, rho float64, rnd func() float64) *VecDense {
 		},
 	}
 }
+
+func BenchmarkVectorSum100000(b *testing.B) { vectorSumBench(b, 100000) }
+
+var vectorSumForBench float64
+
+func vectorSumBench(b *testing.B, size int) {
+	b.StopTimer()
+	a := randVecDense(size, 1, 1.0, rand.NormFloat64)
+	b.StartTimer()
```

Comment:
I still find using `b.RestartTimer()` easier (and dropping `b.StopTimer()` entirely) but ok.